src/__lib__/html.js

```diff
@@ -34,12 +37,32 @@ export default ({
         <body>
             <div id='app'>${html}</div>
             <script>
-            window.gu = ${JSON.stringify({
-                app: {
-                    data,
-                    cssIDs,
-                },
-            })};
+            window.gu = ${sanitiseDomRefs(
+                JSON.stringify({
+                    app: {
+                        data: {
+                            ...data,
+                            content: {
+                                ...Object.keys(data.content).reduce(
+                                    (acc, key) => {
+                                        if (keyRegister.has(key)) {
+                                            acc[
+                                                key
+                                            ] = `document.getElementById('capi-content-${key}-1').innerHTML`;
+                                        } else {
+                                            acc[key] = data.content[key];
+                                        }
+
+                                        return acc;
+                                    },
+                                    {},
+                                ),
+                            },
+                        },
+                        cssIDs,
+                    },
+                }),
+            )};
             </script>
```

Comment:
I've gone with the approach of including `keys` in `window.gu.app.data.content` that have been used when setting up `CapiComponent`s, these `keys` will point to the first instance of a `CapiComponent` in the `dom` which uses this data: eg. `document.getElementById('capi-content-headline-1').innerHTML`. As long as no script that modifies this element runs before the declaration of `window.gu.app.data.content` then this `innerHTML` will be an accurate reflection of the data. The reason I chose to do this rather than omitting it completely from is that it allows the creation of regular 'dynamic' components that use this data.